### DISCUSSION TITLE
CodeQL - UI Based Configuration

### DISCUSSION THREAD

**Discussion by chejn** [community/community#45485] [Code Security] [answered]
### Select Topic Area

Product Feedback

### Body

Hey all,

Regarding the recent addition of UI based set-up for CodeQL,

It would be nice if it was possible to performed advanced configuration of CodeQL through the UI. For example, configuring the languages, query packs and triggers.

I would like to know if this is planned in a future release, otherwise this is a suggestion for a future update.

**Comment by aeisenberg** [comment, marked as answer]
Thank you for your question. I can't make any guarantees about future features and timelines, but I can say that this is an area we are actively exploring.
